OSOE-548: Upgrade to Orchard Core 1.6 in Orchard-Base-Theme

Psichorex opened the pull request:
[OSOE-548](https://lombiq.atlassian.net/browse/OSOE-548)

[OSOE-548]: https://lombiq.atlassian.net/browse/OSOE-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Psichorex committed:
Upgrading to OC 1.6

Piedone commented on Lombiq.BaseTheme/pnpm-lock.yaml:
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
```
See https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/454#discussion_r1180648244

Psichorex replied:
Reverted

Psichorex committed:
Reverting pnpm lock.

Psichorex committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-548

# Conflicts:
#	Lombiq.BaseTheme.Samples/Lombiq.BaseTheme.Samples.csproj
#	Lombiq.BaseTheme/Lombiq.BaseTheme.csproj

Psichorex committed:
Removing AddDataMigration workaround.

Psichorex committed:
Upgrading Lombiq.ChartJs.Tests.UI package.

Psichorex committed:
Updating to latest Lombiq alpha releases.

Psichorex committed:
Removing mistakenly included "v" from versions.

Psichorex committed:
Updating to latest Lombiq.Tests.UI alpha release.

dministro committed:
Upgrading Lombiq.HelpfulLibraries.* package reference version
Upgrading Lombiq.Tests.UI.* package reference version
Upgrading Lombiq.HelpfulExtensions.* package reference version

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Orchard-Base-Theme into issue/OSOE-548

dministro committed:
Merge branch 'issue/OSOE-548' of https://github.com/Lombiq/Orchard-Base-Theme into issue/OSOE-641

Piedone committed:
Merge pull request #69 from Lombiq/issue/OSOE-641

OSOE-641: Fix remaining issues for the Orchard Core 1.6 upgrade in Orchard-Base-Theme

Piedone reviewed (commented):
Just marking the merge from `issue/OSOE-641`.

Piedone committed:
Updating Lombiq.NodeJs.Extensions NuGet to latest alpha

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-548

# Conflicts:
#	Lombiq.BaseTheme/Lombiq.BaseTheme.csproj

Piedone committed:
Updating Lombiq.HelpfulLibraries NuGet reference to latest alpha

sarahelsaig committed:
Fix menu address pointing to admin.

sarahelsaig committed:
Add UI test.

sarahelsaig committed:
use string localizer

sarahelsaig committed:
Extract base theme dependency verification.

sarahelsaig committed:
Add missing Async suffix.

sarahelsaig committed:
Code cleanup.

sarahelsaig committed:
Add missing async suffix.

sarahelsaig committed:
Add comment.

Piedone committed:
Code styling

Piedone committed:
Adding link to GitHub issue

Piedone committed:
Merge pull request #74 from Lombiq/issue/OSOE-647

OSOE-647: Under Orchard Core 1.6, menu items generated via MainMenuNavigationProvider are prefixed with /Admin in Lombiq.BaseTheme

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-548

# Conflicts:
#	Lombiq.BaseTheme.Tests.UI/Lombiq.BaseTheme.Tests.UI.csproj

Piedone committed:
Updating newly added OC package to 1.6

Piedone committed:
Updating Lombiq NuGet references

Piedone committed:
Updating Lombiq NuGet references to release versions
